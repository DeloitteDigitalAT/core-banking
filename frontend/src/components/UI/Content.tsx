import React from "react";

const Content: React.FC = properties => {
  return <div className="px-4 flex-grow">{properties.children}</div>;
};

export default Content;
