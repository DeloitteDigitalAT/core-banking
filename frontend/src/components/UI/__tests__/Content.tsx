import React from "react";
import renderer from "react-test-renderer";
import Content from "../Content";

describe("UI/Content component", () => {
  test("snapshot renders", () => {
    const tree = renderer.create(<Content />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
