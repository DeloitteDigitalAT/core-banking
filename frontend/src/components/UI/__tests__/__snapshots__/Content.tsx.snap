// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI/Content component snapshot renders 1`] = `
<div
  className="px-4 flex-grow"
/>
`;
